### PR TITLE
Add S03E04

### PR DESCRIPTION
**Pull request by stephaniehobson**
Fix bug #1481308

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@jpetto Help! Hugo is convinced that S03E04 is in season 4. It insists the url is `http://localhost:1313/season4/episode04/` Everything else looks okay. Could you try this locally and tell me if you see the same behaviour?

File: content/episodes/2018-08-13-S03E04-virtual-connections.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    Check out this article about [how the internet has changed dating forever](http://blog.mozilla.org/internetcitizen/2018/08/08/how-the-web-changed-dating-forever/).
+    Online dating coach Laurie Davis Edward shares her thoughts on the good, bad and ugly that comes with finding love on the web.
+
+    And, for more about human connection, and what our innate desire for it
+    means for us as we — more and more — love, do business, and find our tribes
+    online, [read this piece by cultural anthropologist Genevieve Bell](http://blog.mozilla.org/internetcitizen/2018/08/13/human-innate-connection-genevieve-bell/).
+
+    Finally, for some bonus audio on [how technology interfered with a marriage
+    proposal](http://blog.mozilla.org/internetcitizen/2018/08/13/online-relationships/) — and commentary on new relationship norms — head over to [Mozilla's blog](http://blog.mozilla.org/internetcitizen/2018/08/13/online-relationships/).
+url: /season4/episode04/
```
Comment (jpetto) [inline content/episodes/2018-08-13-S03E04-virtual-connections.md]: 🙊 

Perhaps a problem with `archetypes/episodes.md`?

Comment (stephaniehobson) [reply content/episodes/2018-08-13-S03E04-virtual-connections.md]: I screwed up typing the file name. I thought I'd found all the places where it was wrong. Sorry to bother you for something so straight forward 😖

Comment (jpetto) [reply content/episodes/2018-08-13-S03E04-virtual-connections.md]: No worries! Bugs like this are the cost of automation. You _assume_ it's fine.

File: content/episodes/2018-08-13-S03E04-virtual-connections.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    Check out this article about [how the internet has changed dating forever](http://blog.mozilla.org/internetcitizen/2018/08/08/how-the-web-changed-dating-forever/).
+    Online dating coach Laurie Davis Edward shares her thoughts on the good, bad and ugly that comes with finding love on the web.
+
+    And, for more about human connection, and what our innate desire for it
+    means for us as we — more and more — love, do business, and find our tribes
+    online, [read this piece by cultural anthropologist Genevieve Bell](http://blog.mozilla.org/internetcitizen/2018/08/13/human-innate-connection-genevieve-bell/).
+
+    Finally, for some bonus audio on [how technology interfered with a marriage
+    proposal](http://blog.mozilla.org/internetcitizen/2018/08/13/online-relationships/) — and commentary on new relationship norms — head over to [Mozilla's blog](http://blog.mozilla.org/internetcitizen/2018/08/13/online-relationships/).
+url: /season3/episode04/
```
Comment (craigcook) [inline content/episodes/2018-08-13-S03E04-virtual-connections.md]: Should this be `episode4` (without the 0) to be consistent with previous episodes?

Comment (stephaniehobson) [reply content/episodes/2018-08-13-S03E04-virtual-connections.md]: Yes, yes it should.

**Comment by stephaniehobson** [comment]
Updated! 🚦

**Pull request merged**